web/src/views/SubmissionPortal/Components/StudyForm.vue

```diff
@@ -65,6 +67,20 @@ export default defineComponent({
       }
     }
 
+    function addDataDoi() {
+      if (studyForm.dataDois === null || studyForm.dataDois.length === 0) {
+        studyForm.dataDois = [{
+          value: '',
+          provider: '',
+        }];
+      } else {
+        studyForm.dataDois.push({
+          value: '',
+          provider: '',
+        });
+      }
```

Comment:
Minor suggestion to prevent having to repeat the new value being added:

```suggestion
      if (!Array.isArray(studyForm.dataDois)) {
        studyForm.dataDois = [];
      }
      studyForm.dataDois.push({
        value: '',
        provider: '',
      });
```